components/BaseComponents.js

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components/native';
+import Colors from '../assets/Colors';
+
+// Text Components
+export const Title = styled.Text`
+  font-family: poppins-medium;
+  font-size: 20px;
+  line-height: 30px;
+  color: ${props => (props.color ? props.color : Colors.black)};
+`;
+
+export const Subhead = styled.Text`
+  font-family: poppins-regular;
+  font-size: 16px;
+  line-height: 24px;
+  color: ${props => (props.color ? props.color : Colors.black)};
+`;
+
+export const Body = styled.Text`
+  font-family: poppins-regular;
+  font-size: 14px;
+  line-height: 20px;
+  color: ${props => (props.color ? props.color : Colors.black)};
+`;
+
+export const Caption = styled.Text`
+  font-family: poppins-medium;
+  font-size: 12px;
+  line-height: 16px;
+  color: ${props => (props.color ? props.color : Colors.black)};
+`;
+
+export const Overline = styled.Text`
+  font-family: poppins-regular;
+  font-size: 14px;
+  line-height: 20px;
+  text-transform: uppercase;
+  color: ${props => (props.color ? props.color : Colors.black)};
+`;
+
+export const TabSelected = styled.Text`
+  font-family: poppins-semibold;
+  font-size: 14px;
+  line-height: 20px;
+  color: ${props => (props.color ? props.color : Colors.black)};
+`;
```

Comment:
Pretty sure we don't need this on Clerk since we don't use that style of tabs like on the customer app.